docs/lake/design.md

```diff
@@ -213,7 +229,7 @@ zed lake commit -p logs <tag-1> <tag-2> <tag-3>
 The commit command also takes an optional title and message that is stored
 in the commit journal for reference.  For example,
 ```
-zed lake commit -p logs -u user@example.com -m "new version of prod dataset" <tag>
+zed lake commit -p logs -user user@example.com -message "new version of prod dataset" <tag>
```

Comment:
I found via black box testing that `-u` and `-m` are not accepted. I'm not opposed to adding them, but for now I've updated this to reflect what works.